README.md

```diff
@@ -16,7 +16,7 @@ git fetch --tags btv-cmssw
 git cms-merge-topic -u cms-btv-pog:FixHistoryBase-v1_from-CMSSW_8_0_12
 git cms-merge-topic -u cms-btv-pog:BoostedDoubleSVTaggerV3-WithWeightFiles-v1_from-CMSSW_8_0_8_patch1
 
-git clone -b 8_0_X_v1.02 --depth 1 https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements.git RecoBTag/PerformanceMeasurements
```

Comment:
Agreed, and I believe we want to update to "-b 8_0_X" right? Or you want to move the tag?